src/main/java/net/minecraftforge/network/NetworkInstance.java

```diff
@@ -102,4 +101,8 @@ boolean isRemotePresent(Connection con) {
         var channels = NetworkContext.get(con).getRemoteChannels();
         return channels.containsAll(ids);
     }
+
+    public BusGroup getNetworkEventBusGroup() {
```

Comment:
the point is to hide this event stuff from modders. honestly killing off all event style networking would probable be better then trying to migrate it